package.json

```diff
@@ -102,7 +102,7 @@
   },
   "dependencies": {
     "@next/bundle-analyzer": "^9.5.1",
-    "@project-r/styleguide": "10.8.0",
+    "@project-r/styleguide": "https://s3.eu-central-1.amazonaws.com/republik-assets-dev/npm/project-r-styleguide-pae-b6870f4.tgz",
```

Comment:
replace with new version number, then` npm i` and add `package.json` and `package-lock.json`

Reply:
Done.